commerce_attribute.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+  commerce_attribute.variation_price_handler:
+    class: Drupal\commerce_attribute\VariationPriceHandler
+    arguments: ['@logger.factory', '@entity.manager', '@entity_type.manager', '@commerce_product.attribute_field_manager', '@entity.query']
```

Comment:
**CHANGE REQUEST**

*Coding standards*

No need to have those super long lines here either:

```
arguments:
  - '@logger.factory'
  - '@entity.manager'
  - '@entity_type.manager'
  - '@commerce_product.attribute_field_manager'
  - '@entity.query'
```

Reply:
Done